devel/external/AEGIS/AEGIS_25.2_training_sample_readme.txt

```diff
@@ -0,0 +1,109 @@
+This README file describes the AEGIS F814W<25.2 training sample, which is a
+randomly ordered sample of galaxies in the AEGIS field with AB magnitude<25.2, in F606W(V) and F814W(I) bands.
+
+NOTE: The columns in the catalog are those expected by galsim.RealGalaxyCatalog.
+However, some parts of that analysis, i.e parametric fits, were not performed.
+Hence those values were manually set and do not have physical meaning.
+
+(1) The catalogs themselves are in AEGIS_galaxy_catalog_[VI]_25.2.fits (where
```

Comment:
I didn't see any instructions about where to get this file.  Maybe this is waiting on @joezuntz to put them up at Manchester?  Should also add these instructions to the [wiki page](https://github.com/GalSim-developers/GalSim/wiki/RealGalaxy%20Data).

We could also consider adding this to the `galsim_download_cosmos` (perhaps renaming this program) as another sample.  `-s aegis` perhaps.  I'm ok with doing this on #795 to make sure it is compatible with any API changes you decide to make there.